hide/comp/SceneEditor.hx

```diff
@@ -365,6 +365,7 @@ class SceneEditor {
 			return true;
 		}
 		tree.get = function(o:PrefabElement) {
+			if(Std.is(o, hrt.prefab.l3d.MeshSpray)) return [];
```

Comment:
@ncannasse wouldn't we want a more generic way to do that ? Something like `Prefab.getDisplayChildren()` or something? Not too fond of having this here